custom_components/energyscore/sensor.py

```diff
@@ -111,9 +111,7 @@ def __init__(self, hass, config):
             LAST_UPDATED: None,
         }
 
-        entity_reg = er.async_get(hass)
-        if entity_reg.async_get(self._price_entity).platform == "nordpool":
-            self._nordpool = True
+        self.hass = hass
```

Comment:
What is this for?

Reply:
to access hass variable in process_data method